package.json

```diff
@@ -89,9 +89,6 @@
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-2": "^6.18.0",
-    "danger": "^3.6.0",
```

Comment:
This was causing issues with Flow for me. We aren't using danger-ci atm so we can remove it (hopefully just for now. it would be nice to get it back)